qt/RpcClient.h

```diff
@@ -33,6 +34,16 @@ extern "C"
   struct tr_session;
 }
 
+struct RpcResponse
```

Comment:
Missing default constructor, results in asserts because of uninitialized/random `networkError` value in e.g. RpcQueue.cc:73.